_components/form-controls/07-range-input.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Range input
+parent: Form controls
+maturity: alpha
+order: 07
+lead: The range slider allows users to choose an approximate number from a range.
```

Comment:
`range slider` should be consistent with the component name? In this case, maybe `range input`?

Reply:
Updated in 57dd76050ada879e3103f7324f19c1bfb013d463.